src/runner/index.js

```diff
@@ -377,11 +377,17 @@ export default class Runner extends EventEmitter {
         return this;
     }
 
-    screenshots (path, takeOnFails, pattern) {
+    screenshots (...options) {
+        let [path, takeOnFails, pathPattern, fullPage] = options;
```

Comment:
What about this pattern for parameters?

```js
screenshots(path, {
     takeOnFails,
     pathPattern,
     fullPage
})
```

Because of  the `path` parameter is required and other parameters are optional.

Reply:
I do not want to parse `path` as object in this situation:
```JS
screenshots({  path: 'kekeke', takeOfFails: true, pathPattern: 'pathPattern', fullPage: true })
```
In this case path === object, so we should write something like this:
```JS
if (typeof path === 'object')
   { path, takeOfFails, fullPage } = path;
```
or
```JS
path = path.path
```
I think these kinds of naming are confusing

Reply:
Ok